tests/test_df_transformer.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from optimus.spark import get_spark
+import optimus as op
+
+class TestDataFrameTransformer(object):
+
+    def test_lower_case(self):
+        source_data = [
+            ("BOB", 1),
+            ("JoSe", 2)
+        ]
+        source_df = get_spark().createDataFrame(
+            source_data,
+            ["name", "age"]
+        )
+
+        transformer = op.DataFrameTransformer(source_df)
+        actual_df = transformer.lower_case("*").get_data_frame
+
+        expected_data = [
+            ("bob", 1),
+            ("jose", 2)
+        ]
+        expected_df = get_spark().createDataFrame(
+            expected_data,
+            ["name", "age"]
+        )
+
+        assert(expected_df.collect() == actual_df.collect())
+
+    def test_upper_case(self):
```

Comment:
This should be a class method. 

```
@classmethod
def test_upper_case(self):
```

Reply:
The [pytest examples use instance methods](https://docs.pytest.org/en/latest/getting-started.html).  Looks like class methods work, so that's what I added, but I don't think class methods are standard.  Not sure - I'm still a Python newbie.